src/lib/cozy-client/DataAccessFacade.js

```diff
@@ -18,7 +22,13 @@ export default class DataAccessFacade {
   }
 
   setup(cozyUrl, options) {
```

Comment:
Shall we change the call of `setup(cozyUrl, options)` to be sure the `cozyUrl` contains a protocol?